distributed/protocol/compression.py

```diff
@@ -152,7 +152,7 @@ def maybe_compress(
     min_size=10_000,
     sample_size=10_000,
     nsamples=5,
-    compression=dask.config.get("distributed.comm.compression"),
```

Comment:
I think this is something that needs to be set globally before the process starts and so should be a default kwarg here